### ISSUE TITLE
`io.udash.css.CssView` missing from `udash-css`

### ISSUE THREAD

**Issue by kamilkloch** [UdashFramework/udash-core#288]
`io.udash.css.CssView` is present in `udash-css_sjs` but missing from `udash-css`.
As a result, I am unable to use `udash-css` in the backend project. Is this desired? Any hints?

**Comment by ddworak** [comment]
`_sjs` JARs are dedicated to Scala.js only and contain platform-specific code. SBT will automatically choose the right JAR for the project when using `%%%` (see https://www.scala-js.org/doc/project/dependencies.html). 

As for `udash-css`, there's a regular JVM dependency which requires all the code you need for backend CSS rendering. The style definitions are cross-compiled and available both in JVM and JS JARs. 

Please refer to sample project generated by: `sbt new UdashFramework/udash.g8` to see that in action.

**Comment by kamilkloch** [comment]
Thanks for the response. I know the mentioned template. I just tried to add udash-css styling to ScalaTags and I failed. CssView is missing for JVM and so is the implicit conversion to Modifier. What am I missing? Thanks.

**Comment by ddworak** [comment]
Binding a CSS style to an HTML element in a view is basically just adding a class name to it. The `CssView` utilities help with doing that on DOM elements (a `Modifier` is defined as a function applied to one of such) which are obviously only avaiable in the browser. Are you doing server-side HTML rendering?

**Comment by kamilkloch** [comment]
Yes, I was trying to render the page on the server side. ScalaTags work fine, I wanted use udash for CSS.

**Comment by kamilkloch** [comment]
Probably a dumb question, perhaps you can clarify. In backend, this works:
```scala
a(style := "color: red", href := "/button", h4("Button Example")),
```
but using `udash-css` does not. 
```scala
a(GlobalStyles.btn, href := "/button", h4("Button Example")),
```

Why?

**Comment by ddworak** [comment]
Not at all, it's quite valid. We just haven't expected this use case, but it's quite simple to do - I've actually just created a PR (https://github.com/UdashFramework/udash-core/pull/289) with some of our platform-independent utilities shared on the JVM as well. Basically you only need a way to convert a style to `scalatags.Text.all.Modifier`, which in the PR looks like this:
```scala
trait CssText {
  implicit def style2TextMod(s: CssStyle): Modifier = new CssText.TextStyleModifier(s)
  implicit def styles2TextMod(s: CssStyle*): Modifier = new CssText.TextStyleModifier(s: _*)
}

object CssText extends CssText {
  private class TextStyleModifier(styles: CssStyle*) extends Modifier {
    override def applyTo(t: Builder): Unit =
      styles.foreach { s =>t.appendAttr("class", Builder.GenericAttrValueSource(s.classNames.mkString(" ", " ", ""))) }
  }
}
```
You can either use this snippet or wait for a few days for RC6 with this in our shared part of the code.

Thanks for pointing this out!

**Comment by ddworak** [comment]
Closed with 0.8.0-RC6 released today.
